src/pwa-buildpack/project-setup/create-configuration-files/index.md

```diff
@@ -0,0 +1,268 @@
+                     test: /\.css$/,
+                     use: [
+                         'style-loader',
+                         {
+                             loader: 'css-loader',
+                             options: {
+                                 importLoaders: 1
+                             }
+                         }
+                     ]
+                 }
+             ]
+         },
+         resolve: await MagentoResolver.configure({
+             paths: {
+                 root: __dirname
+             }
+         }),
+         plugins: [
+             new MagentoRootComponentsPlugin(),
+             new webpack.NoEmitOnErrorsPlugin(),
+             new webpack.EnvironmentPlugin({
+                 NODE_ENV: env.mode,
+                 SERVICE_WORKER_FILE_NAME: 'sw.js'
+             })
+         ]
+
+    };
+
+    return config;
+}
+```
+
+This configuration sets up Webpack for your development environment.
+Some important things to note in this configuration:
+
+* How it configures the `MagentoResolver` service
+* The inclusion of the `MagentoRootComponent` as a plugin
+* The use of `webpack.EnvironmentPlugin` to pass environment variables
+
+### Add development mode configuration
+
+Add the following development mode configuration before returning the `config` object:
+
+``` javascript
+if (env.mode === "development") {
+    config.devServer = await PWADevServer.configure({
+        publicPath: process.env.MAGENTO_BACKEND_PUBLIC_PATH,
+        backendDomain: process.env.MAGENTO_BACKEND_DOMAIN,
+        serviceWorkerFileName: process.env.SERVICE_WORKER_FILE_NAME,
+        paths: themePaths,
+        id: 'magento-my-theme'
+    });
+
+    // A DevServer generates its own unique output path at startup. It needs
+    // to assign the main outputPath to this value as well.
+
+    config.output.publicPath = config.devServer.publicPath;
+
+    config.plugins.push(
+         new ServiceWorkerPlugin({
+             env,
+             paths: themePaths,
+             enableServiceWorkerDebugging: false,
+             serviceWorkerFileName: process.env.SERVICE_WORKER_FILE_NAME
+         })
+     );
+
+     config.plugins.push(
+         new webpack.HotModuleReplacementPlugin()
+     );
+} else {
+  throw Error('Only "development" mode is currently supported. Please pass "--env.mode development" on the command line.');
+}
+```
+
+This code block does the following:
+
+* Create a `PWADevServer` confgiuration object and attach it to the Webpack configuration object.
+* Create a `ServiceWorkerPlugin` and attach it to the Webpack configuration object.
+* Add a `webpack.HotModuleReplacementPlugin` to enable fast workflow.
+* Configure Webpack to throw an error if you are not in development mode.
+
+### Add start script
+
+Edit the `scripts` section of your `package.json file` so it looks like the following:
+
+``` javascript 
+"scripts": {
+    "start" : "webpack-dev-server --progress --color --env.mode development",
```

Comment:
The `env.mode` in this command must also be replaced with `env.phase`. And in the example below!